assets/js/blocks/customer-account/editor.scss

```diff
@@ -1,5 +1,12 @@
 .wc-block-editor-customer-account {
 	padding: em($gap-smaller) em($gap-smaller);
+
+	&.aligncenter {
+		text-align: center;
+		div {
+			display: inline-block;
+		}
+	}
```

Comment:
The `aligncenter` class is already being applied to the block and it looks like WordPress/WooCommerce has some properties that are responsible to center the block based on the margins: 
<img width="370" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/20469356/f8123163-7647-4616-8e5f-b8aa356c5cd5">

Looking at the width property of the block and how it reacts to each alignment option, I noticed that when using the Align Left or Align Right, the width is limited to the content size of the block (basically the icon and the "My Account" text); while when using the Align Center, the width is limited to the max content size of the page (650px):

https://github.com/woocommerce/woocommerce-blocks/assets/20469356/64459aaf-1f07-42dc-bee7-5df75a7ab62f

Considering that, I'm wondering if instead of changing the 'text-align' and 'display' properties, we could just limit the width to the content size of the block (the icon and the text) and match the same behavior as the Align Left and Align Right options:

```suggestion
	&.aligncenter {
		width: fit-content; // or max-content
	}
```

What do you think?

Reply:
I don't think IE supports `fit-content` as seen here https://caniuse.com/mdn-css_properties_width_fit-content

Also another issue I noticed is that aligning `left/right`, the block will collapse because it is using `floats` and there is no `clear` directive. This is also happening in `trunk` so it's a separate concern. Perhaps we can switch to using `flexbox` instead of `floats`.

Reply:
I think it is safe to use `fit-content` (or 'max-content'), because WooCommerce dropped support to IE a few years ago: https://developer.woocommerce.com/2021/05/24/developer-advisory-plan-to-remove-ie-11-support-in-woocommerce/

Reply:
Right, I totally forgot we dropped support.